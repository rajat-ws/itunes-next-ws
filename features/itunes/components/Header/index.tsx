import { colors, media } from "@app/themes";
import { injectIntl } from "react-intl";
import styled from "styled-components";

const HeaderContainer = styled.div`
  && {
    display: flex;
    justify-content: center;
    width: 50%;
    margin: 0 auto;
    background: ${colors.primary};

    ${media.lessThan("mobile")`
    width: 90%;
    `}
  }
`;

const HeaderTitle = styled.span`
  && {
    font-size: 2rem;
    color: white;
  }
`;

const Header = () => {
  return (
    <HeaderContainer>
      <HeaderTitle> Itunes Store 🎶 </HeaderTitle>
    </HeaderContainer>
  );
};

export default injectIntl(Header);
