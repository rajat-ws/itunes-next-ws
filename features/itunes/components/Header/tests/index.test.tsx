import { render } from "@utils/testUtils";
import Header from "../index";

describe("<Header />", () => {
  it("should render and match the snapshot", () => {
    const { baseElement } = render(<Header />);

    expect(baseElement).toMatchSnapshot();
  });
});
