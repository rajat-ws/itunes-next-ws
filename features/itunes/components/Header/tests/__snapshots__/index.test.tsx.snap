// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render and match the snapshot 1`] = `
<body>
  <div>
    <div
      class="Header__HeaderContainer-sc-12p05nx-0 cMXrEl"
    >
      <span
        class="Header__HeaderTitle-sc-12p05nx-1 eeGxed"
      >
         Itunes Store 
      </span>
    </div>
  </div>
</body>
`;
